peepsView.test.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const fs = require('fs');
+const PeepsView = require('./peepsView');
+const PeepsModel = require('./peepsModel');
+
+describe('PeepsView class', () => {
+  beforeEach(() => {
+    document.body.innerHTML = fs.readFileSync('./index.html');
+    model = new PeepsModel;
+  })
+
+  it('displays peeps', () => {
```

Comment:
this only tests that you display a single peep, would it not be worth testing for multiple peeps?